src/main.cpp

```
@@ -5,31 +5,39 @@
 #include <iostream>
 
 #include <QApplication>
+#include <QDir>
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)